keras_maskrcnn/bin/evaluate_coco.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+"""
+Copyright 2017-2018 Fizyr (https://fizyr.com)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import os
+import sys
+
+import keras
+import keras.preprocessing.image
+import tensorflow as tf
+
+# Allow relative imports when being executed as script.
+if __name__ == "__main__" and __package__ is None:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+    import keras_maskrcnn.bin
+    __package__ = "keras_maskrcnn.bin"
+
+# Change these to absolute imports if you copy this script outside the keras_retinanet package.
+from ..preprocessing.coco import CocoGeneratorMask
+from ..utils.coco_eval import evaluate_coco
+from ..models.resnet import custom_objects
+from keras_retinanet.utils.keras_version import check_keras_version
+
+def get_session():
+    config = tf.ConfigProto()
+    config.gpu_options.allow_growth = True
+    return tf.Session(config=config)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(description='Simple training script for COCO object detection.')
```

Comment:
Hmm this is probably also wrong in retinanet, but let's correct it here first of all ;)